diff --git a/recipes/qt/6.x.x/conanfile.py b/recipes/qt/6.x.x/conanfile.py
index b55a2ef88..52d2a4275 100644
--- a/recipes/qt/6.x.x/conanfile.py
+++ b/recipes/qt/6.x.x/conanfile.py
@@ -287,8 +287,8 @@ class QtConan(ConanFile):
             raise ConanInvalidConfiguration("apple-clang >= 13.1 is required by qt >= 6.6.1 cf QTBUG-119490")
 
         if Version(self.version) >= "6.8.3":
-            if self.settings.compiler == "msvc" and Version(self.settings.compiler.version) < "193":
-                raise ConanInvalidConfiguration("Visual Studio 2022 (MSVC 1930 or newer) is required by qt >= 6.8.3")
+            if self.settings.compiler == "msvc" and Version(self.settings.compiler.version) < "192":
+                raise ConanInvalidConfiguration("Visual Studio 2019 (MSVC 1920 or newer) is required by qt >= 6.8.3")
 
             if self.settings.compiler == "apple-clang" and Version(self.settings.compiler.version) < "15":
                 raise ConanInvalidConfiguration("apple-clang >= 14 is required by qt >= 6.8.3")
